bin/tasks.js

```diff
@@ -4,12 +4,12 @@ const path = require('path');
 const cwd = process.cwd();
 const target = path.resolve(cwd, 'public');
 const source = path.resolve(__dirname, '../src');
-const sass = require('witch')('node-sass');
+const npmsass = require('npm-sass');
 
 // jscs:disable maximumLineLength
 module.exports = {
   'make-folders': `mkdir -p ${target}/js ${target}/css ${target}/images`,
-  'compile-css': `${sass} ${source}/scss/app.scss ${target}/css/app.css --include-path ./node_modules`,
+  'compile-css': `${npmsass} ${source}/scss/app.scss > ${target}/css/app.css`,
```

Comment:
In fact, since this isn't an npm script, you'll need the fully qualified path to the binary, so I'd leave as it was with `witch` finding the binary path.